ktor-server/ktor-server-plugins/ktor-server-rate-limit/jvmAndNix/src/io/ktor/server/plugins/ratelimit/RateLimit.kt

```diff
@@ -14,7 +14,7 @@ internal val LOGGER = KtorSimpleLogger("io.ktor.server.plugins.ratelimit.RateLim
 internal val LIMITER_NAME_GLOBAL = RateLimitName("KTOR_GLOBAL_RATE_LIMITER")
 internal val LIMITER_NAME_EMPTY = RateLimitName("KTOR_NO_NAME_RATE_LIMITER")
 
-internal data class ProviderKey(
+public data class ProviderKey(
```

Comment:
Could we move tests in ktor-server-rate-limit with the `testApplication` and make it internal?